curvefs/src/client/s3/client_s3_adaptor.cpp

```diff
@@ -62,6 +62,7 @@ S3ClientAdaptorImpl::Init(
     chunkFlushThreads_ = option.chunkFlushThreads;
     maxReadRetryIntervalMs_ = option.maxReadRetryIntervalMs;
     readRetryIntervalMs_ = option.readRetryIntervalMs;
+    objectPrefix_ = option.objectPrefix;
```

Comment:
maybe objectPrefix should be a member of FsInfo just like FSType?

Reply:
objectPrefix is inited in the [FuseopInit](https://github.com/opencurve/curve/blob/master/curvefs/src/client/fuse_s3_client.cpp#L63) process  ` ::curvefs::client::common::S3Info2FsS3Option(s3Info, &fsS3Option);` And s3Info just come from FsInfo.